packages/i18nify-js/src/modules/core/data/countryToDefaultLocales.ts

```diff
@@ -0,0 +1,98 @@
+// A map of countries to their default locale.
+export const COUNTRY_TO_DEFAULT_LOCALES = {
```

Comment:
No need to have another config, utilise `COUNTRY_TO_ALL_LOCALES` and keep the default locale at 0th index for all countries. Create the config dynamically from there.